rails/7/files/.erb-lint.yml

```diff
@@ -0,0 +1,42 @@
+---
+glob: "**/*.{html,text,js}{+*,}.erb"
+exclude:
+  - '**/vendor/**/*'
+EnableDefaultLinters: true
+linters:
+  ErbSafety:
+    enabled: true
+    better_html_config: .better-html.yml
+  PartialInstanceVariable:
+    enabled: true
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Layout/ArgumentAlignment:
+        EnforcedStyle: with_fixed_indentation
+        IndentationWidth: 2
+      Layout/InitialIndentation:
+        Enabled: false
+      Layout/LineLength:
+        Enabled: true
+        Max: 100
+      Layout/TrailingEmptyLines:
+        Enabled: false
+      Layout/LeadingEmptyLines:
+        Enabled: false
+      Layout/TrailingWhitespace:
+        Enabled: false
+      Naming/FileName:
+        Enabled: false
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Style/TrailingCommaInArguments:
+        EnforcedStyleForMultiline: no_comma
+      Style/TrailingCommaInHashLiteral:
+        EnforcedStyleForMultiline: comma
+      Lint/UselessAssignment:
+        Enabled: true
+      Rails/OutputSafety:
+        Enabled: true
```

Comment:
Question, @nialbima  - are these rules coming directly from EEC, and are there any that we might *generally* consider worthwhile that aren't used in EEC? (for instance, that variable naming one you mentioned in DM's - it makes sense for EEC to turn that off, but I think we can generally say that rule is worthwhile by default). 

Curious how much of this is sanitized from EEC

Reply:
Additionally: We may need to do some work to integrate this fully into the Gnarly template. Lets circle back on this when we both have more cycles :D